ci/Jenkinsfile

```diff
@@ -135,7 +138,7 @@ pipeline {
       steps { timeout(90) { script {
         def flags = []
         if (params.TEST_NAME)  { flags.add("-k=${params.TEST_NAME}") }
-        if (params.TEST_SCOPE) { flags.add("-m=${params.TEST_SCOPE}") }
+        if (params.TEST_SCOPE_FLAG) { flags.add(params.TEST_SCOPE_FLAG) }
```

Comment:
Why are we passing the whole flag instead of just the value? Why did we change this? And if we change it, why did we not change `TEST_NAME`? Seems silly to me.

Reply:
Kinda confusing if you ask me, but okay.